src/content/docs/help/space-access-control.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Space Access Control"
+description: "How to control who has access to your space."
+---
+
+import { Aside } from '@astrojs/starlight/components';
+
+Operately makes it easy to control who can see and work in your spaces. We believe in transparency, so by default, anyone in your company can view what's happening in a space.
+
+You can change who can access your space via the **Manage Access** section of any space.
+
+### Understanding access levels
+
+You can set different access levels for:
+
+- People on the internet
```

Comment:
Let's remove this part line.
I've opened an issue to address it https://github.com/operately/operately/issues/1874.